resources/auth/auth.py

```diff
@@ -0,0 +1,82 @@
+import jwt
+import os, json
+
+from flask import abort, request, g
+from flask_restful import abort, Resource
+from flask_restful.reqparse import RequestParser
+from passlib.apps import custom_app_context as pwd_context
+
+from resources.api import app, db
+from resources.models import Users
+
+JWT_PASS = os.environ["SECRET_KEY"]
+JWT_ALGORITHM = "HS256"
+
+
+def verify_user(username, password):
+    """
+    checks for password given against password_hash to verify user
+    """
+    user = Users.query.filter_by(username=username).first()
+
+    if user and pwd_context.verify(password, user.password):
+        g.user_id = user.user_id
+
+        return user
+
```

Comment:
too many blank lines. See [this](https://google.github.io/styleguide/pyguide.html#Blank_Lines) for a detailed guide.